watertap/flowsheets/full_water_resource_recovery_facility/test/test_BSM2_P_extension.py

```diff
@@ -56,71 +56,71 @@ def test_solve(self, system_frame):
             0.24219, rel=1e-3
         )
```

Comment:
Would it make sense to check the condition number at the flowsheet level between old scaling approach and new?

Reply:
Then, every time another PR impacts scaling at this flowsheet level, we can track whether condition number improves or not as we go.